core/templates/dev/head/domain/subtopic_viewer/subtopic-viewer-backend-api.service.ts

```diff
@@ -19,10 +19,29 @@
 require('domain/utilities/url-interpolation.service.ts');
 require('domain/subtopic_viewer/subtopic-viewer-domain.constants.ajs.ts');
 
+import { SubtopicDataObjectFactory } from
+  'domain/subtopic_viewer/SubtopicDataObjectFactory';
+import { SubtitledHtmlObjectFactory } from
+  'domain/exploration/SubtitledHtmlObjectFactory';
+import { SubtopicPageContentsObjectFactory } from
+  'domain/topic/SubtopicPageContentsObjectFactory';
+import { RecordedVoiceoversObjectFactory } from
```

Comment:
Add this in alphabetically sorted order i.e, above line 22.

Reply:
Done